edx/edx-ora2
# Add a new button to edit an ORA in Studio

**TL;DR -** Add a new button to edit an ORA in Studio. This is more helpful than the generic "View in Studio" button which loses context of the current subsection and also allows staff to jump to studio from the Instructor Dashboard ORA tab which previously had no direct link to studio.

<img width="564" alt="Screen Shot 2021-05-28 at 14 53 53" src="https://user-images.githubusercontent.com/12944465/120029819-9937d380-bfc4-11eb-8f08-01ac680bc8c9.png">

JIRA: [EDUCATOR-5736](https://openedx.atlassian.net/browse/EDUCATOR-5736)

**Developer Checklist**


**Testing Instructions**

_In-course ORA_
1. As staff/admin/instructor, visit an ORA in LMS
3. Verify new button appears as in screenshot, "View ORA in Studio"
3. Click the button
4. Verify you are redirected to the matching ORA in studio

_Instructor Dashboard_
**Note:** have to first enable the XBlock view API by adding `FEATURES['ENABLE_XBLOCK_VIEW_ENDPOINT'] = True` in `edx-platform/lms/envs/devstack.py`.

1. As staff/admin/instructor, visit the "Instructor" tab for a course in the LMS
2. Click to "Open Responses" sub-tab
3. Click on a linked Open Response, this renders the block in the same tab
3. Verify new button appears as in screenshot, "View ORA in Studio"
4. Verify you are redirected to the matching ORA in studio

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:


FYI: @edx/masters-devs-gta
